src/extension.ts

```diff
@@ -41,54 +41,41 @@ const formatText = (text: string) => `\r${text.split(/(\r?\n)/g).join("\r")}\r`;
 function handleExit(code: number | null, signal: string | null) {
 	var actionHint: string = '\tPlease report this issue at https://github.com/probe-rs/probe-rs/issues/new';
 	if (code) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with an unexpected code: ".concat(`${code}`).concat(actionHint));
 	} else if (signal) {
-		vscode.window.showErrorMessage("ERROR: `probe-rs-debugger` exited with signal: ".concat(`${signal}`).concat(actionHint));
+		vscode.window.showErrorMessage("ERROR: `probe-rs-debug` exited with signal: ".concat(`${signal}`).concat(actionHint));
 	}
 }
 
 // Messages to be sent to the debug session's console. 
-// Anything sent before or after an active debug session is silently ignored by VSCode. 
-// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'INFO' , 'WARN', `DEBUG`, or `TRACE` to match the RUST log behaviour. 
-// Any local messages that start with `CONSOLE` will ALWAYS be logged.
-// Any messages that come from the `probe-rs-debugger` STDERR will always be logged, and will already conform with the RUST LOG setting.
+// Any local (generated directly by this extension) messages MUST start with 'ERROR', or 'probe-rs-debug' , or `DEBUG`. 
+// Any messages that start with `ERROR` or `probe-rs-debug` will ALWAYS be logged.
+// Any messages that come from the `probe-rs-debug` STDERR will always be logged.
 function logToConsole(consoleMesssage: string, fromDebugger: boolean = false) {
 	console.log(consoleMesssage); // During VSCode extension development, this will also log to the local debug console
 	if (fromDebugger) {
-		// Any messages that come directly from the debugger, are assumed to be RUST_LOG messages and should be logged to the console.
-		vscode.debug.activeDebugConsole.appendLine(consoleMesssage);
-		// The one exception is RUST_LOG messages of the `error` variant. These deserve to be shown as an error message in the UI also.
+		// RUST_LOG messages of the `error` variant. These deserve to be shown as an error message in the UI also.
```

Comment:
```suggestion
		// STDERR messages of the `error` variant. These deserve to be shown as an error message in the UI also.
```

Reply:
Did you mean stderr? With RUST_LOG? The variable name might not stay the same :)

Reply:
Yes, I did ... good catch :)